src/racer/ast_types.rs

```diff
@@ -159,6 +160,8 @@ impl Ty {
             },
             LitKind::FloatUnsuffixed(_) => make_match(PrimKind::F32),
             LitKind::Bool(_) => make_match(PrimKind::Bool),
+            // TODO: support err
+            LitKind::Err(_) => None,
```

Comment:
The new kind added with [this PR](https://github.com/rust-lang/rust/pull/57651).

Reply:
We don't have no item to complete for bad literal like `'aaa'`, so I don't think we need TODO comment here.